app/.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
#155 mentions ignoring package-lock.json as well.  I'm not sure exactly why, since we are committing package-lock in this repo